src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -303,7 +303,7 @@ const useGroupInfo = (channelId = null) => {
     }
 
     refresh('channelList');
-    refresh('chat');
+    refreshWithId('chat', channelId);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to line 256, the `refresh` function call has been replaced with `refreshWithId('chat', channelId)`. This change appears to make the refresh operation more targeted, which could potentially improve performance by reducing unnecessary data fetching or re-rendering. However, it's important to ensure that the `channelId` provided always corresponds to an existing chat in the database.



<!-- This is an auto-generated comment by OSS CodeRabbit -->